lib/endpoints/vms.js

```diff
@@ -52,6 +52,19 @@ var VALID_VM_ACTIONS = [
 var DEFAULT_LIST_VM_LIMIT = common.MAX_LIST_VMS_LIMIT;
 var DEFAULT_LIST_VM_OFFSET = 0;
 
+// How often vmapi will check CNAPI to determine if a given server (and thus
+// whether the vm) is running.
+var SERVER_STATUS_CACHE_UPDATE_SECONDS = 30;
```

Comment:
Can we call these `const` at some point?

Reply:
Sure - the VMAPI version of node (v6.17) certainly supports that.